muesli/models.py

```diff
@@ -371,7 +371,11 @@ def getResultsForStudent(self, student):
         for points in pointsQuery.all():
             results[points.exercise_id] = {'points': points.points,
                                            'exercise': points.exercise}
-        results['sum'] = sum([x for x in [r['points'] for r in list(results.values())] if x])
+        nonNullPoints = [x for x in [r['points'] for r in list(results.values())] if x]
```

Comment:
I think we dont need to create a list out of `result.values()` here (yeah I know, the code was already there, lets improve it) 😊